app-doc/rnote/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=rnote
+PKGDES="Vector-based drawing app for sketching and handwritten notes"
+PKGSEC=doc
+PKGDEP="gtk-4 glib libadwaita poppler gstreamer alsa-lib"
+BUILDDEP="meson cargo llvm cmake git gtk-update-icon-cache"
```

Comment:
Please remove `meson cargo git` per style guidelines.

Reply:
Actually, replace `cargo` with `rustc` as it is the actual package behind `cargo`.